### PR TITLE
[New] [Author] Add rsuite@3.0.2 w/ npm auto-update

### PR DESCRIPTION
**Pull request by hiyangguo**
Pull request for issue: #12712
Related issue(s): #12713

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): [rsuite/rsuite][git-repository-url]
 * Official website (optional, not the repository): [http://rsuitejs.com/][homepage]
 * NPM package url (optional): [npmjs.com/package/rsuite][npm]
 * License and its reference:[MIT][license]
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 36
   - Count of stars: 903
   - Count of forks: 63
 * NPM download stats (optional):
   - Downloads in the last day: 
   - Downloads in the last week: 355
   - Downloads in the last month:

# Essential checklist
 * [x] I'm the author of this library
   * [x] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [ ] Close corresponding issue in commit message
 * [ ] Mention related issue(s), people in commit message, comment.

[license]:https://github.com/rsuite/rsuite/blob/master/LICENSE
[git-repository-url]: https://github.com/rsuite/rsuite
[homepage]:http://rsuitejs.com/
[npm]:https://www.npmjs.com/package/rsuite

### CODE REVIEW

File: ajax/libs/rsuite/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "rsuite",
+  "filename": "rsuite.min.js",
+  "version": "3.0.1",
+  "description": "A suite of react components",
+  "homepage": "http://rsuitejs.com/",
```
Comment (sashberd) [inline ajax/libs/rsuite/package.json]: Lets use https

Comment (sashberd) [inline ajax/libs/rsuite/package.json]: Lets use https

Comment (hiyangguo) [reply ajax/libs/rsuite/package.json]: We only need to host the version after 3.0.0,So how should I configure it? AndI checked the 3.0.0 and later versions. Their directories are the same as my current configuration.Thank you very much.

Comment (sashberd) [reply ajax/libs/rsuite/package.json]: You pattern already fetches only version 3.0.0 and above. Lets wait for @sufuf3 @PeterDaveHello @extend1994 answer to my question above

Comment (hiyangguo) [reply ajax/libs/rsuite/package.json]: @sashberd  OK,Thank you very much.
@sufuf3 @PeterDaveHello @extend1994 Can you help me?

Comment (extend1994) [reply ajax/libs/rsuite/package.json]: @hiyangguo Yes, it's alright not to add the versions the library author or main  contributor(s) think unnecessary.
By the way, could you tell if the file `dist/suite.js` of 0.0.16 and the below versions is compatible with a browser? I think the answer is yes, but it's better to check with the one who is more familiar with the library.
Once this is confirmed and url is updated to https, the PR can be merged! :slightly_smiling_face:

Comment (hiyangguo) [reply ajax/libs/rsuite/package.json]: @extend1994  I am this library main contributor , I'm sure that before 3.0.0 version of this lib is not required for hosting. And I will update later, thank you very much.

**Comment by hiyangguo** [comment]
@extend1994 I have changed it,Could you please help me to merge?

**Comment by hiyangguo** [comment]
@extend1994 Sorry, waiting for me check these files later and PR again.

**Comment by hiyangguo** [comment]
@extend1994  What should I do ?
image

**Comment by extend1994** [comment]
@hiyangguo Updating the permission to 644 would solve that problem.

**Comment by hiyangguo** [comment]
@extend1994 It's ok.Thank you.

**Comment by hiyangguo** [comment]
@extend1994 Hello , could you help me to merge this PR now ?

**Comment by hiyangguo** [comment]
@PeterDaveHello @dakshshah96 @sufuf3 @sashberd @extend1994
Hello, anybody can help me? Thank you so much.

File: ajax/libs/rsuite/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "react",
+    "rsuite",
+    "component",
+    "react-component"
+  ],
+  "author": "Simon Guo <simonguo.2009@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:rsuite/rsuite.git"
```
Comment (sufuf3) [inline ajax/libs/rsuite/package.json]: Let's revise to `git://github.com/rsuite/rsuite.git`
We provide the public URL to let people access the link.
People need to add the ssh public key to GitHub, then he(she) can use the current URL.
Many thanks.

**Comment by hiyangguo** [comment]
@sufuf3 I have changed. Could you help me recheck agin? Thank you.

**Pull request merged**